lib/ts/recipe/emailpassword/types.ts

```diff
@@ -403,6 +405,7 @@ export type FormBaseProps<T> = {
 
     clearError: () => void;
     onError: (error: string) => void;
+    onFetchError?: (err: Response) => void;
```

Comment:
what is this for?

Reply:
This is called when we get a non-200 status code, used to handle 403 errors returned by sign in/up endpoints.